services/main-frontend/src/components/page-specific/organizations/index/components/OrganizationBanner.tsx

```diff
@@ -57,8 +65,82 @@ const OrganizationBanner: React.FC<Props> = ({ organization }) => {
         `}
       >
         <OrganizationIcon />
-        <OrganizationText name={organization.name} />
-        <OrganizationSelectButton slug={organization.slug} />
+        <OrganizationText name={organization.name} hidden={organization.hidden} />
+        <div
+          className={css`
+            display: flex;
+            align-items: center;
+            gap: 0.5rem;
+            margin-left: auto;
+            margin-right: 1rem;
+          `}
+        >
+          <OnlyRenderIfPermissions
+            action={{ type: "create_courses_or_exams" }}
+            resource={{ type: "organization", id: organization.id }}
+          >
+            <button
+              type="button"
+              onClick={() => router.push(manageOrganizationRoute(organization.id))}
```

Comment:
If you use an onclick handler, you cannot for example ctrl-click or middle mouse button click to open in a new tab. Therefore it would be better to just use a link (with next/link) and style that to look like a button.